src/ipa/rpi/controller/rpi/clock_recovery.h

```diff
@@ -0,0 +1,47 @@
+#include <chrono>
+#include <list>
+using namespace std::chrono_literals;
+
+namespace RPiController {
+
+class ClockRecovery
+{
+public:	
+	std::list<int64_t> error_values;
```

Comment:
I'm afraid all data member names are going to have to go camelCase. And with an underscore.  So `errorValues_` here. Sorry (again)...